packages/vats/src/lib-chainStorage.js

```diff
@@ -113,3 +113,46 @@ export async function makeStorageNode(chainStorage, childName) {
   return storageNode;
 }
 harden(makeStorageNode);
+
+/**
+ * For testing, creates a chainStorage root node over an in-memory map
+ * and exposes both the map and the sequence of received messages.
+ *
+ * @param {string} rootPath
+ */
+export const makeTransientStorageKit = rootPath => {
```

Comment:
@gibson042 suggested "transient" may not sound non-production enough. could be `makeFakeStorageKit`. Before renaming I'd like to get clear on the taxonomy of storage kinds with @michaelfig